autofs-5.1.0-beta1 - fix ldap default schema config

From: Ian Kent <ikent@redhat.com>

If there is no schema set in the autofs configuration then the ldap
lookup module assumes it will get a NULL return when it asks for the
default schema.

So don't set the default schema at defaults initialization time.
---
 CHANGELOG      |    1 +
 lib/defaults.c |   25 -------------------------
 2 files changed, 1 insertion(+), 25 deletions(-)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -204,6 +204,7 @@
 - add amd map format parser.
 - fix wildcard key lookup.
 - fix out of order amd timestamp lookup.
+- fix ldap default schema config.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/lib/defaults.c
+++ autofs-5.0.5/lib/defaults.c
@@ -301,31 +301,6 @@ static int conf_load_autofs_defaults(voi
 	if (ret == CFG_FAIL)
 		goto error;
 
-	ret = conf_update(sec, NAME_MAP_OBJ_CLASS,
-			  DEFAULT_MAP_OBJ_CLASS, CONF_ENV);
-	if (ret == CFG_FAIL)
-		goto error;
-
-	ret = conf_update(sec, NAME_ENTRY_OBJ_CLASS,
-			  DEFAULT_ENTRY_OBJ_CLASS, CONF_ENV);
-	if (ret == CFG_FAIL)
-		goto error;
-
-	ret = conf_update(sec, NAME_MAP_ATTR,
-			  DEFAULT_MAP_ATTR, CONF_ENV);
-	if (ret == CFG_FAIL)
-		goto error;
-
-	ret = conf_update(sec, NAME_ENTRY_ATTR,
-			  DEFAULT_ENTRY_ATTR, CONF_ENV);
-	if (ret == CFG_FAIL)
-		goto error;
-
-	ret = conf_update(sec, NAME_VALUE_ATTR,
-			  DEFAULT_VALUE_ATTR, CONF_ENV);
-	if (ret == CFG_FAIL)
-		goto error;
-
 	ret = conf_update(sec, NAME_APPEND_OPTIONS,
 			  DEFAULT_APPEND_OPTIONS, CONF_ENV);
 	if (ret == CFG_FAIL)
